### ISSUE TITLE
stopAnimation() problem

### ISSUE THREAD

**Issue by nicolasdelfino** [oblador/react-native-animatable#76]
Hi, thanks for the great lib, I´m having a problem stopping function animations like:

```
//click btn
click() { 
       this.refs.view.stopAnimation()
       this.refs.view.tada(1000);
}
```

Is this not supported?

**Comment by oblador** [comment]
Triggering a new animation will automatically stop the previous so in this case it has no real effect. Also remember that these functions are async so triggering it in rapid succession not a good idea. I'm guessing your case is slightly more advanced than this, what are you trying to do?

**Comment by oblador** [comment]
Looking at https://github.com/oblador/react-native-animatable/issues/63, this might actually be a bug. Can you report back and I'll try to fix it? Btw, Kim B says hello 👋 Small world 😸

**Comment by nicolasdelfino** [comment]
**Wow it really is, tell her I said hi!** 

I realized I described my issue poorly, what I was trying to do was to actually stop and reset the animated objects when fast-clicking through a navigation control, only animating the arrows when the user stayed idle on a slide to avoid animation stacking. When I realized that the animations were called async I just had to do a  setInterval wired to a mutex to check idle time before calling the animation and voila.

TLDR: not Animatable´s fault, just me forgetting that the functions were called async.

Thanks for your help!

**Comment by oblador** [comment]
Amaze 👍 If you think this this behaviour should be mentioned in the docs, a PR would be appreciated. Closing this for now.
